static_src/stores/route_store.js

```diff
@@ -40,6 +42,26 @@ class RouteStore extends BaseStore {
     return !!route.app_guid;
   }
 
+  getRouteURLForApp(app) {
+    const allRoutes = this.getAll();
+    const boundRoutes = allRoutes.filter((route) => route.app_guid === app.guid);
```

Comment:
To bind two apps to a single route? Yeah, that is possible. That's how the blue/green [autopilot cf plugin](https://github.com/contraband/autopilot#method) works.

I currently have this setup in case you want to test. https://adborden-node.apps.cloud.gov/ routes to `adborden-node` and `adborden-nodejs-venerable` in the cf-deck-testing space.